gprofiler/profilers/node.py

```diff
@@ -186,6 +186,7 @@ def create_debugger_socket(nspid: int, ns_link_name: str) -> WebSocket:
     sock.settimeout(10)
     _validate_ns_node(sock, ns_link_name)
     _validate_pid(nspid, sock)
+    logger.info(f"Created debugger socket for pid: {nspid} (nodejs)")
```

Comment:
Prefer to log the host PID and not NS PID. The NSPID is not unique and can (and will) repeat across containers, so by reading the log you don't truly know which process it's referring to. Please log the PID itself (if not available in the function, you can pass it from a higher-level function which does have it)

Reply:
It's helpful to log the NSPID as well since you have it here. I'm thinking of what could be useful when we debug failures here, and the NSPID could help as well, but PID itself is required

Reply:
I added host PID to NSPID in that line, but in _changed_dso I log only host PID